packages/web/src/components/tracks-table/TracksTable.tsx

```diff
@@ -316,8 +316,7 @@ export const TracksTable = ({
         track.is_delete || track._marked_deleted || !!track.user?.is_deactivated
       const isOwner = track.owner_id === userId
       const isUnlisted = track.is_unlisted
-      const isFavorited = track.has_current_user_saved
-      if (isLocked || deleted || isOwner || (isUnlisted && !isFavorited)) {
+      if (isLocked || deleted || isOwner || isUnlisted) {
```

Comment:
undid change from previous PR based on last product sync.

same deal in change below on line 355